src/test/java/com/faforever/integration/TestDatabase.java

```diff
@@ -0,0 +1,53 @@
+package com.faforever.integration;
+
+import com.faforever.api.clan.ClanMembershipRepository;
+import com.faforever.api.clan.ClanRepository;
+import com.faforever.api.client.OAuthClientRepository;
+import com.faforever.api.player.PlayerRepository;
+import com.faforever.api.user.UserRepository;
+import lombok.Getter;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+import static org.junit.Assert.assertEquals;
+
+@Component
+@Getter
+public class TestDatabase {
+  private ClanRepository clanRepository;
+  private UserRepository userRepository;
+  private ClanMembershipRepository clanMembershipRepository;
+  private PlayerRepository playerRepository;
+  private OAuthClientRepository oAuthClientRepository;
```

Comment:
This is needed to reduce the method parameters for all factories as long as https://github.com/FAForever/faf-java-api/issues/71 is not fixed ...
Otherwise the Factories can use `@Inject` to grab the repositories ...